README.md

```diff
@@ -331,7 +331,7 @@ Auth::hasAnyScope(['scope-f', 'scope-k']) // false
 
 # Acting as a Keycloak user in tests
```

Comment:
Since this is not the main title, we can change it to H2.

Reply:
Updated the README. Do you think this can be merged followed by release of a new version into packagist this week? I assume it is possible since there is not much change request around the PR, but not sure about your release schedules.

Reply:
I don't know about release schedule but since it's not a major version, it can be released ASAP.